src/app/home/home.component.html

```diff
@@ -26,14 +26,6 @@
               <input value="{{ entry.value }}" class="input" type="text" disabled>
           </p>
         </div>
-        <div class="field is-grouped panel-heading">
-          <p class="control">
-            <a class="button is-primary" (click)=start()> Start </a>
-          </p>
-          <p class="control">
-            <a class="button is-danger" (click)=stop()> Stop </a>
-          </p>
-        </div>
```

Comment:
can we now delete this component completely?

Reply:
yes